1 Presentation layer/CandidateEvaluator.Client/Pages/Question/CreateQuestion.cshtml

```diff
@@ -0,0 +1,89 @@
+﻿@page "/category/{categoryId:guid}/question/new"
+@using Microsoft.AspNetCore.Blazor.Services
+@inject Services.UserIdentityService Http
+@inject IUriHelper UriHelper
+
+@if (_categoryModels == null)
+{
+    <p><em>Loading...</em></p>
+
+}
+else
+{
+    <a href="/category/@_questionModel.CategoryId/questions" class="btn">Go back</a>
+    <div class="form-group">
+        <label>Category</label>
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown">
+                @if (_selectedCategory == null)
+                {
+                    <span>Select category</span>
+                }
+                else
+                {
+                    <span>@_selectedCategory.Name</span>
+
+                }
+            </button>
+            <div class="dropdown-menu">
+                @foreach (var category in _categoryModels)
+                {
+                    <button class="dropdown-item" onclick="@(c => _setSelectedCategory(category))">@category.Name</button>
+                }
+            </div>
+        </div>
+    </div>
+    <div class="form-group">
+        <label>Name</label>
+        <input class="form-control" type="text" bind=@_questionModel.Name placeholder="Enter question name..." />
+    </div>
+    <div>
+        <textarea class="form-control" rows="4" bind=@_questionModel.Text placeholder="Enter question" />
+    </div>
+    <div>
+        <button class="btn btn-blue" onclick="@Add">Add</button>
+    </div>
+
+}
+
+@functions
+{
+[Parameter]
+Guid CategoryId { get; set; }
+
+private List<Contract.Models.Category> _categoryModels;
```

Comment:
Put a namespace in a using

Reply:
unfortunately there is an ambiguity with current namespace and e.g Pages.Question.Question.cshtml and solution does not build with namespace specified explicity